oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyStateFactory.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.jackrabbit.oak.plugins.document;
+
+import org.apache.jackrabbit.oak.api.PropertyState;
+import org.apache.jackrabbit.oak.commons.Compression;
+
+public class DocumentPropertyStateFactory {
+
+    public static PropertyState createPropertyState(DocumentNodeStore store, String name, String value, Compression compression) {
```

Comment:
what about adding a variant that doesn't ask for the `Compression`? in that case it would perhaps hard code to GZIP, or later that could be configured..

Reply:
Done